cmd/minishift/cmd/start.go

```diff
@@ -220,7 +220,7 @@ func runStart(cmd *cobra.Command, args []string) {
 		MachineName:      constants.MachineName,
 		Ip:               ip,
 		Port:             constants.APIServerPort,
-		RoutingSuffix:    getDefaultRoutingPrefix(ip),
+		RoutingSuffix:    configCmd.GetDefaultRoutingSuffix(ip),
```

Comment:
@gbraad  This will ignore flag `--routing-suffix string            Default suffix for the server routes.` right? If yes, -1 for this change.

Reply:
NVM, I saw  cmd/minishift/cmd/config/functions.go and it is fine.

Reply:
it does exactly the same...

Reply:
This has to be changed anyway when we refactor the config, as this is actuall allinstances config, or instance-specific.